src/libs/TransactionUtils/index.ts

```diff
@@ -683,7 +684,21 @@ function getMerchantOrDescription(transaction: OnyxEntry<Transaction>) {
  * Return the list of modified attendees if present otherwise list of attendees
  */
 function getAttendees(transaction: OnyxInputOrEntry<Transaction>): Attendee[] {
-    return transaction?.modifiedAttendees ? transaction.modifiedAttendees : (transaction?.comment?.attendees ?? []);
+    const attendees = transaction?.modifiedAttendees ? transaction.modifiedAttendees : (transaction?.comment?.attendees ?? []);
+    if (attendees.length === 0) {
+        const details = getPersonalDetailByEmail(currentUserEmail);
+        attendees.push({
+            email: currentUserEmail,
+            login: details?.login ?? currentUserEmail,
+            displayName: details?.displayName ?? currentUserEmail,
+            accountID: currentUserAccountID,
+            text: details?.displayName ?? currentUserEmail,
+            searchText: details?.displayName ?? currentUserEmail,
+            avatarUrl: details?.avatarThumbnail ?? '',
+            selected: true,
+        });
+    }
```

Comment:
If no attendee is preset then the default attendee should be the expense creator and not necessary the current user

Coming from https://github.com/Expensify/App/issues/65428